cocos/ui/UISlider.h

```diff
@@ -265,6 +265,11 @@ class CC_GUI_DLL Slider : public Widget
      */
     float getZoomScale()const;
 
+    Sprite* getSlidBallNormalRenderer();
+    Sprite* getSlidBallPressedRenderer();
+    Sprite* getSlidBallDisabledRenderer();
+    Node* getSlidBallRenderer();
```

Comment:
I have updated the pull request to declare these functions as const.

The SlidBallNormalRenderer, SlidBallPressedRenderer and SlidBallDisabledRenderer are declared as Sprites but SlidBallRenderer is declared as a Node (it is only used as a parent for the other Sprites).